source/process/developer.rst

```diff
@@ -75,10 +75,11 @@ Both declined and accepted candidates will be invited to share feedback on their
 - **Application/Email - Review of code samples** - In `your application <https://jobs.lever.co/mattermost/>`__ we welcome you to include links to any public open source work you've done, including GitHub, GitLab or BitBucket profiles. If you'd like to share privately we can send you our ID for GitHub, GitLab or BitBucket, or we can review code you send in via email. If you really want to impress us, `make a contribution to our open source project by completing a Help Wanted ticket. <../developer/contribution-guide.html>`__
 - **Email - Follow-up questions** - If your code samples look good, we'll email you some questions specific to the role.
```

Comment:
I find a lot of candidates aren't submitting code samples -- should we emphasize this point, or relax it?

Reply:
@lieut-data good to hear this - suggests I'm the not the only person who thinks this isn't particularly realistic here. I'll update to be more in line with what *actually* happens in this stage too.